dashboard/app/helpers/levels_helper.rb

```diff
@@ -345,13 +345,18 @@ def match_t(text)
 
   def level_title
     if @script_level
-      if @script_level.script.legacy_curriculum?
-        "#{data_t('game.name', @game.name)} #{'#' + @script_level.position.to_s unless @script_level.position == 1}"
+      script = data_t_suffix('script.name', @script_level.script.name, 'title')
+      stage = @script_level.name
+      position = @script_level.position
+      if @script_level.script.stages.many?
+        "#{script}: #{stage} ##{position}"
```

Comment:
Ordering within the title is fine, as far as I know. The only thing that might be weird is R2L, but looking at production it reversed it. So looks good.